library/README.md

```diff
@@ -0,0 +1,15 @@
+# CLT Library
+
+This library allows to enable or disable CLT in the Linux kernel by using [generic netlink](https://docs.kernel.org/networking/generic_netlink.html) to interact with the kernel.
+
+## Usage
+
+The implementations **must** provide 2 methods:
+- One to **enable** CLT on a given socket with given trace and span IDs.
+- One to **disable** CLT on a given socket.
+
+## Existing implementations
+
+At the moment, implementations are available in the following languages:
+- [Python](./python/)
+- [Go](./golang/)
```

Comment:
In [demo/docker/athos/Dockerfile](https://github.com/Advanced-Observability/cross-layer-telemetry/blame/506fa5db34d330fe59002e647c7f176d3b1498ff/demo/docker/athos/Dockerfile#L30), we need to apply the following:

```diff
- RUN wget https://raw.githubusercontent.com/Advanced-Observability/cross-layer-telemetry/main/clt_genl.py
+ RUN wget https://raw.githubusercontent.com/Advanced-Observability/cross-layer-telemetry/main/library/python/clt_genl.py
```